docs/Changelog.md

```diff
@@ -14504,3 +14504,66 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to high-precision and 8 bit numeric tensors.</dd>
 </dl>
 
+### <a name="SoftmaxCrossEntropyLoss-12"></a>**SoftmaxCrossEntropyLoss-12**</a>
+
+  Loss function that measures the softmax cross entropy
+  between 'scores' and 'labels'.
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), where N is the batch_size
+  
+  shape(scores): (N, C) where C is the number of classes, or (N, C, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  shape(labels): (N) where each value is 0 <= labels[i] <= C-1, or (N, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  
+  The loss for one sample, l_i, can caculated as follows:
+      l_i = -y[i][c][d1][d2]..[dk], where i is the index of classes.
+  or
+      l_i = -y[i][c][d1][d2]..[dk]*weights[c], if 'weights' is provided.
+  
+  where:
+      p = Softmax(scores)
+      y = log(p)
+      c = labels[i][d1][d2]...[dk]
+  
+  Finally, L is reduced:
+  L = ReduceSum(L), if reduction = 'sum';
+      ReduceMean(L), if reduction = 'mean'; if "weight" is provided, output is averaged by sum of weights.
```

Comment:
I think `weight` also applies to the `sum` and `none` mode. The description here seems saying `weight` only applies to `mean`. In addition, the equation to compute weighted loss is unclear. Is it L_weighted = Mul(L, weight)?

Reply:
yes, but when there is a weighted mean, then mean is not averaged by the number of elements in the output but by the sum of weights

Reply:
above there is a description that applies to all reduction types and in case of reduction=mean it has to be averaged by sum of weights as well

Reply:
Sorry. So there are two behaviors? What are equations of them?

Reply:
For mean, yes. If weights is provided it's divided by sum of weights otherwise by the number of elements in output.